ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -59,6 +59,11 @@
   private static final String KSQL_WEBSOCKETS_NUM_THREADS_DOC =
       "The number of websocket threads to handle query results";
 
+  public static final String COMMAND_RETRY_LIMIT_CONFIG = "command.retry.limit";
```

Comment:
If not retrying indefinitely may cause servers to diverge, why are we making it configurable? :D

Reply:
I want to avoid bugs inadvertently causing tests to hang, especially in Jenkins. So hence the config, which I've set to a small number for tests. Note that even if this gets set to something small in real usage, the result is that the command thread will exit - the server's state will still be consistent.